File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.js
function solution(str1, str2) {
  let result = '';

  // str1과 str2의 길이는 같으므로 하나를 기준으로 반복
  for (let i = 0; i < str1.length; i++) {
    result += str1[i] + str2[i]; // str1의 i번째 문자와 str2의 i번째 문자를 번갈아 추가
  }

  return result;
}

File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.cs
using System;

public class Solution {
    public string solution(string str1, string str2) {
        string newString = "";

        for (int i = 0; i < str1.Length; i++)
        {
            newString += str1[i].ToString() + str2[i].ToString();
        }

        return newString;
    }
}